.github/dependabot.yml

```diff
@@ -497,10 +501,6 @@ updates:
     directory: "/receiver/sapmreceiver"
     schedule:
       interval: "weekly"
-  - package-ecosystem: "gomod"
-    directory: "/receiver/scrapererror"
-    schedule:
-      interval: "weekly"
```

Comment:
Was this also deprecated?

Reply:
It was removed, I just run `make gendependabot`